test/pages/register-a-beacon/activity.test.tsx

```diff
@@ -37,4 +46,64 @@ describe("Activity", () => {
 
     expect(screen.getByLabelText(/rowing/i)).toBeChecked();
   });
+
+  describe("the page heading", () => {
+    function assertPageHeadingContains(
+      expected: string,
+      environment: Environment,
+      purpose: Purpose
+    ): void {
+      render(
+        <ActivityPage
+          form={activityFormTestData}
+          flattenedRegistration={{
+            environment,
+            purpose,
+          }}
+        />
+      );
+
+      expect(screen.getByText(expected, { exact: false })).toBeDefined();
+    }
+
+    it("should not show the purpose if the environment is land", () => {
+      assertPageHeadingContains(
+        "select the land activity",
+        Environment.LAND,
+        Purpose.PLEASURE
+      );
+    });
```

Comment:
Really like use of unit tests for this.  Feel like I've been too quick to reach for Cypress to test things that could have been unit tested.  👍🏻